### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sul-devops-team**

### CODE REVIEW

File: spec/features/item_registration_spec.rb [hunk 1]
```diff
@@ -72,7 +72,7 @@
       'workflow_id' => 'goobiWF',
       'label' => 'object title',
       'tag' => ['Process : Content Type : Book (ltr)', 'tag : test', "Registered By : #{user.sunetid}"],
-      'rights' => 'default',
+      'rights' => 'world',
```
Comment (mjgiarlo) [inline spec/features/item_registration_spec.rb:75]: @sul-dlss/infrastructure-team any ideas why I'd need to make this change to fix the build? FWIW, I was seeing this in `main` as well prior to switching into this branch. Did a default change? Or does Argo need more cocina-related changes? cc: @jcoyne @justinlittman

**Comment by peetucket** [comment]
Maybe because of #2394 which I started late in the day Friday. Have not had a chance to look at why the test would change though.

________________________________

**Comment by peetucket** [comment]
Though actually #2394 is not merged yet so should be independent of this test failure in the dependency updates. Not sure what's going on here.

________________________________

**Review by mjgiarlo — APPROVED** [review]
Approved by automated merge script

**Pull request merged**